src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -605,5 +605,240 @@ private class TransferModeFeature : ITransferModeFeature
         {
             public TransferMode TransferMode { get; set; }
         }
+
+        private static class Log
+        {
+            private static readonly Action<ILogger, string, int, Exception> _preparingNonBlockingInvocation =
+                LoggerMessage.Define<string, int>(LogLevel.Trace, new EventId(1, nameof(PreparingNonBlockingInvocation)), "Preparing non-blocking invocation of '{target}', with {argumentCount} argument(s).");
```

Comment:
Something @rynowak pointed out in another PR is that we shouldn't actually use `nameof` because this name is a public API of sorts. Using `nameof` means that when we rename the private function that triggers this message we could accidentally rename this public API name.